tests/config/state/tests.yml

```diff
@@ -1,83 +1,106 @@
-    operations: [ "set", "get", "delete", "bulkget", "bulkset", "bulkdelete", "transaction", "etag",  "first-write", "ttl" ]
+    operations: [ "transaction", "etag",  "first-write", "ttl" ]
   - component: azure.tablestorage.storage
     allOperations: false
-    operations: ["set", "get", "delete", "etag", "bulkset", "bulkdelete", "first-write"]
+    operations: [ "etag", "first-write"]
+    config:
+      # This component requires etags to be in this format
+      badEtag: "W/\"datetime'2023-05-09T12%3A28%3A54.1442151Z'\""
   - component: azure.tablestorage.cosmosdb
     allOperations: false
-    operations: ["set", "get", "delete", "etag", "bulkset", "bulkdelete", "first-write"]
+    operations: [ "etag", "first-write"]
+    config:
+      # This component requires etags to be in this format
+      badEtag: "W/\"datetime'2023-05-09T12%3A28%3A54.1442151Z'\""
   - component: oracledatabase
     allOperations: false
-    operations: [ "set", "get", "delete", "bulkset", "bulkdelete", "transaction", "etag",  "first-write", "ttl" ]
+    operations: [ "transaction", "etag",  "first-write", "ttl" ]
   - component: cassandra
     allOperations: false
-    operations: [ "set", "get", "delete", "bulkset", "bulkdelete", "ttl" ]
+    operations: [ "ttl" ]
   - component: cloudflare.workerskv
     allOperations: false
     # Although this component supports TTLs, the minimum TTL is 60s, which makes it not suitable for our conformance tests
-    # TODO: perhaps create a special case `ttl60` operation for this component
-    # where the test would set a TTL of 60s for this particular operation.
-    # `ttl` and `ttl60` are mutually exclusive, and `allOperations` would
-    # exclude `ttl60`
-    operations: [ "set", "get", "delete", "bulkset", "bulkdelete"]
+    operations: []
   - component: cockroachdb
     allOperations: false
-    operations: [ "set", "get", "delete", "bulkget", "bulkset", "bulkdelete", "transaction", "etag", "query", "ttl" ]
+    operations: [ "transaction", "etag", "first-write", "query", "ttl" ]
+    config:
+      # This component requires etags to be numeric
+      badEtag: "9999999"
   - component: rethinkdb
     allOperations: false
-    operations: [ "set", "get", "delete", "bulkset", "bulkdelete"]
+    operations: []
   - component: in-memory
     allOperations: false
-    operations: [ "set", "get", "delete", "bulkget", "bulkset", "bulkdelete", "transaction", "etag",  "first-write", "ttl" ]
+    operations: [ "transaction", "etag",  "first-write", "ttl" ]
   - component: aws.dynamodb.docker
     allOperations: false
-    operations: [ "set", "get", "delete", "etag", "bulkset", "bulkdelete", "transaction", "first-write" ]
+    operations: [ "transaction", "etag", "first-write" ]
   - component: aws.dynamodb.terraform
     allOperations: false
-    operations: [ "set", "get", "delete", "etag", "bulkset", "bulkdelete", "transaction", "first-write" ]
+    operations: [ "transaction", "etag", "first-write" ]
```

Comment:
Does it mean that only the non-required operations are now listed here? This is different from the other conformance tests. Is it a decision to change all conf tests to work this way? If so, this schema does not work well because it should be renamed to "optionalOperations" and allOperations should be removed.